recipes-kernel/linux/files/5.x-kernel/0001-iot2050-add-iot2050-platform-support.patch

```diff
@@ -673,7 +673,6 @@ index 000000000000..5ffcbd9641b7
 +	pinctrl-0 = <&main_mmc1_pins_default>;
 +	ti,driver-strength-ohm = <50>;
 +	disable-wp;
-+	no-1-8-v;// for SR1 board,needed to be removed for sr2
```

Comment:
Nack. Design the DTS to be directly usable for both boards - and the patch to be submittable upstream.

Reply:
I'm not sure upstream kernel is not ready for sr2.
I suggest we should reconstruct the dts when we are planing unify kernel for both boards then.
currently,we split two images one for sr1 and the other for sr2. so this change I think is reasonable.

Reply:
Always work towards upstream, even if not submitting yet. Do not start splitting the development for PG1 and PG2 by created DTs that only work for one.